### PR TITLE
data: rahulpandey70

### PR DESCRIPTION
**Pull request by rahulpandey70**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3239"></a>

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Please fix your JSON profile file naming. It should be `{your GitHub username}.json` -> `rahulpandey70.json`. Otherwise, features will not be functional.

If you need any help feel free to ask.

**Review by rahulpandey70 — COMMENTED** [review]
Rename file name data/RahulPandey.json to data/rahulpandey70

File: data/rahulpandey70.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      "name": "Github",
+      "url": "https://github.com/rahulpandey70",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://instagram.com/_imrahulpandey",
+      "icon": "FaInstagram"
+    },
```
Comment (Pradumnasaraf) [inline data/rahulpandey70.json:22]:
```suggestion
    }
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thanks for the changes.

I just saw the JSON syntax is wrong. There is an extra `,`. Please remove that.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**